### DISCUSSION TITLE
Where do we ask questions about this?

### DISCUSSION THREAD

**Discussion by Allistah** [alandtse/alexa_media_player#1040] [General]
Where is the best place to ask questions about this app?  I have it installed and would like to be able to configure things through the UI but am not sure how to go about it.  I want to configure messages for Alexa to announce and then use those messages in Automations.  Right now I have to write everything in a script and then refer to them in the automations by using script.announce.my.stuff.

Thanks

**Comment by alandtse** [comment]
This is probably the best place but we just opened it. The main thread is probably the most active though.
https://community.home-assistant.io/t/echo-devices-alexa-as-media-player-testers-needed/58639/

Perhaps you can post your question and ask people to respond here.
